_posts/2020-03-31-new-release.md

```diff
@@ -0,0 +1,6 @@
+---
+title:  "Tableau Connector SDK for 2019.4 Released, latest version of SDK now targets 2020.1"
+abstract: "The Connector SDK for 2019.4 has released for those wishing to target that version. The main SDK branch now targets Tableau 2020.1"
+---
+
+On March 11th, with the full release of Tableau 2020.1 we moved the latest version of the master branch to target Tableau 2020.1. This means that the samples and connectors made with this version of the SDK may not work with older versions of Tableau. To develop against Tableau 2019.4, please download the "Tableau Connector SDK for Tableau 2019.4" release.
```

Comment:
If possible can you do a markdown link for "Tableau Connector SDK for Tableau 2019.4" and remove quotes.

Reply:
can also make that a link to the release maybe?

Reply:
Added link, and a bit on vendor attributes as well